PyMca5/__init__.py

```diff
@@ -47,8 +48,18 @@
     import ctypes
     from ctypes.wintypes import MAX_PATH
 
-if os.path.exists(os.path.join(\
-    os.path.dirname(os.path.dirname(__file__)), 'bootstrap.py')):
```

Comment:
I could have simply commented out this `ImportError`, but adding a conditional check of the environment variable `ALLOW_PYMCA_IMPORT` is more robust.

* It does not break existing behavior
* It enables tests to import the PyMca module
* It flexibly allows the behavior to be changed from the environment or CLI